packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -90,14 +90,17 @@ class _DropdownMenuPainter extends CustomPainter {
 
 // Do not use the platform-specific default scroll configuration.
 // Dropdown menus should never overscroll or display an overscroll indicator.
+// The default scrollbar platforms will apply.
```

Comment:
It will use the scrollbars defined in ScrollBehavior, but it should be using the ones from the behavior in the context, right?